sdk/azidentity/azidentity_test.go

```diff
@@ -23,6 +23,13 @@ func Test_AuthorityHost_Parse(t *testing.T) {
 
 func Test_SetEnvAuthorityHost(t *testing.T) {
 	err := os.Setenv("AZURE_AUTHORITY_HOST", envHostString)
+	defer func() {
+		// Unset that host environment vairable to avoid other tests failed.
```

Comment:
Nit: variable

Reply:
It was my own misspelling but good to catch

Reply:
Fixed